java/src/main/java/com/google/cloud/dataproc/templates/databases/README.md

```diff
@@ -89,6 +89,54 @@ To query using default catalog name -:
 ```
 You can replace the ```casscon``` with your catalog name if it is passed. This is an important step to query the data from Cassandra. Additional details on usage of query can be found in this [link](https://github.com/datastax/spark-cassandra-connector/blob/master/doc/14_data_frames.md).
 
+## Executing Cassandra to Bigquery Template
+### General Execution
+
+```
+export REGION=<gcp-region>
+export GCP_PROJECT=<gcp-project-name>
+export GCS_STAGING_LOCATION=<gcs-staging-location>
+export JOB_TYPE=SERVERLESS 
+export SUBNET=<dataproc-serverless-subnet>
+
+bin/start.sh \
+-- --template CASSANDRATOBQ \
+--templateProperty cassandratobq.input.keyspace=<keyspace-name> \
+--templateProperty cassandratobq.input.table=<input-table-name> \
+--templateProperty cassandratobq.input.host=<cassandra-host-ip> \
+--templateProperty cassandratobq.bigquery.location=<dataset>.<table-name> \
+--templateProperty cassandratobq.output.mode=<Append|Overwrite|ErrorIfExists|Ignore> \
+--templateProperty cassandratobq.temp.gcs.location=<gcs-bucket-name> 
+```
+### Example Submission:
+```
+export REGION=us-central1
+export GCP_PROJECT=myproject
+export GCS_STAGING_LOCATION=gs://staging
+export JOB_TYPE=SERVERLESS 
+export SUBNET=projects/myproject/regions/us-central1/subnetworks/default
+
+bin/start.sh \
```

Comment:
I would recommend to use following in this example submission:
1. Absolute / meaningful values instead of `<cassandra-host-ip>`, `table` and `keyspace`
2. Update the example showing CQL as `select * from employee.salaries;`